ludwig/typing.py

```diff
@@ -0,0 +1,28 @@
+"""Common typing for Ludwig dictionary parameters."""
+from typing import Any, Dict
+
+# Dictionary of parameters used to configure an input or output feature.
+# https://ludwig.ai/latest/configuration/features/supported_data_types/
+FeatureConfigDict = Dict[str, Any]
+# Dictionary representation of the ModelConfig object.
+# https://ludwig.ai/latest/configuration/
+ModelConfigDict = Dict[str, Any]
```

Comment:
2 newlines between definitions, to visually separate them.

Reply:
Done.